modern/src/map/Map.js

```diff
@@ -93,6 +93,23 @@ const Map = ({ children }) => {
     };
   }, []);
 
+  useEffect(() => {
+    if (mapReady && Object.values(map.getStyle().sources).some(e => e.type === 'raster')) {
+      const onIdle = function () {
+        const zoom = Math.round(map.getZoom());
+        if (zoom !== map.getZoom()) {
+          map.zoomTo(zoom);
+        }
+      };
+      map.scrollZoom.setWheelZoomRate(1);
+      map.on('idle', onIdle);
+      return () => {
+        map.scrollZoom.setWheelZoomRate(1/450);
```

Comment:
Can we save the original zoom instead of hardcoding this value?

Reply:
This is not the zoom value but the rate how fast mouse wheel will zoom. It is not something that users can change.

Reply:
I understand, but where did you get the value from? What if developers change it in the future?

Reply:
`scrollZoom` class has a private property called `_wheelZoomRate`. I can use that to access the original value but what if developers change that?

Reply:
Interesting. No getter? I guess let's leave it as is.